tower_cli/cli/transfer/send.py

```diff
@@ -935,23 +935,23 @@ def import_extra_credentials(self, existing_object, new_creds):
                 tower_cli.get_resource('job_template').disassociate_credential(
                     existing_object['id'], existing_name_to_id[cred]
                 )
-                self.log_change("Removed extra credential {}".format(cred))
+                self.log_change("Removed credential {}".format(cred))
             except TowerCLIError as e:
-                self.log_error("Unable to remove extra credential {} : {}".format(cred, e))
+                self.log_error("Unable to remove credential {} : {}".format(cred, e))
 
-        # Creds to add is the difference between existing_creds and extra_creds
+        # Creds to add is the difference between existing_creds and creds
         for cred in list(set(new_creds).difference(existing_creds)):
             try:
                 new_credential = tower_cli.get_resource('credential').get(**{'name': cred})
             except TowerCLIError as e:
-                self.log_error("Unable to resolve extra credential {} : {}".format(cred, e))
+                self.log_error("Unable to resolve credential {} : {}".format(cred, e))
                 continue
 
             try:
                 tower_cli.get_resource('job_template').associate_credential(existing_object['id'], new_credential['id'])
```

Comment:
It looks like `associate_credential` still uses the extra_credentials endpoint.

https://github.com/ansible/tower-cli/blob/ed8546b88fab6b07566fa1c2dc3385c82aa85e34/tower_cli/resources/job_template.py#L145

I would expect that https://github.com/ansible/tower-cli/pull/610 would still be necessary for this to work